src/functions/template-tags/general.php

```diff
@@ -682,10 +696,14 @@ function tribe_meta_event_archive_tags( $label = null, $separator = ', ', $echo
 
 		$terms = get_the_terms( get_the_ID(), 'post_tag' );
 
-		if ( is_wp_error( $terms ) ) {
+		if ( empty( $terms ) || is_wp_error( $terms ) ) {
```

Comment:
At this point (3rd time's the charm?) I think I'd functionalize this logic.\ 🤔

Reply:
I'm not sure that would be a good trade-off. It could get bloated over time and introduce side-effects as the `tec_parse_terms_list` function is added more functionality and parameters.